monkey/monkey_island/cc/ui/src/components/logo/VersionComponent.js

```diff
@@ -15,7 +15,7 @@ class VersionComponent extends AuthComponent {
   }
 
   componentDidMount() {
-    this.authFetch('/api/island/version')
+    this.authFetch('/api/island/version', {}, true)
```

Comment:
This isn't an action that the user explicitly takes.
```suggestion
    this.authFetch('/api/island/version', {}, false)
```